README.md

```diff
@@ -71,6 +73,28 @@ See [browserslist/browserslist](https://github.com/browserslist/browserslist) fo
 :bulb: You can also define browsers in a [separate browserslist file](https://github.com/browserslist/browserslist#config-file)
 
 ## Adding Polyfills
+
+#### v3
+
+Add polyfills to the settings section of your eslint config. Append the name of the object and the property (if it exists). No need to include the `prototype` if it is an instance:
```

Comment:
I’d caution against allowing the prototype to be omitted; there’s always the chance of a static and prototype method being named the same.

Reply:
Good call. This plugin can't detect if arbitrary variables are instances or not so I omitted this. Making this distinction will be useful for future scenarios where we add support for detecting if a variable is an instance

Reply:
But also, if support for an instance method foo, and a static method foo, isn’t consistent in every browser.